DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -223,68 +196,67 @@ function Set-TargetResource
     Update-LocationTagInApplicationHostConfigForAuthentication -WebSite $EndpointName -Authentication "basic"
     Update-LocationTagInApplicationHostConfigForAuthentication -WebSite $EndpointName -Authentication "windows"
         
-
     if ($IsBlue)
     {
         Write-Verbose "Set values into the web.config that define the repository for BLUE OS"
-        #PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbprovider" -value $eseprovider
-        #PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbconnectionstr"-value $esedatabase
-        #ESE database is not present in current build
         PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbprovider" -value $jet4provider
         PSWSIISEndpoint\Set-AppSettingsInWebconfig -path $PhysicalPath -key "dbconnectionstr" -value $jet4database
```

Comment:
OLE is not supported on blue core. The original code for setting the eseprovider for blue was correct.

this is the matrix I believe we should follow:
Win 2008 R2 Core –no support
Win 2008 R2 Full – OLE
Windows 2012 (Win8) Core – OLE? (not sure about this one)
Windows 2012 (Win8) Full – OLE
Windows 2012 R2 (Blue) Full – ESENT
Windows 2012 R2 (Blue) Core – ESENT
Threshold Server Core –ESENT
Threshold Server Full -ESENT